src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -346,6 +346,45 @@ object DataElementDiscoveryUtils {
       AuditReportConstants.ELEMENT_DISCOVERY_NODE_TYPE
     )
   }
+
+  def appendSqlNodesToWorkbook(xtocpg: Try[Cpg], workbookResult: ListBuffer[List[String]]): Unit = xtocpg match {
+    case Success(cpg) =>
+      val sqlNodes = cpg.sqlColumn.l
+      for (sqlNode <- sqlNodes) {
+        val path       = sqlNode.file.name.headOption.getOrElse(AuditReportConstants.AUDIT_EMPTY_CELL_VALUE)
+        val lineNumber = sqlNode.lineNumber.getOrElse(AuditReportConstants.AUDIT_EMPTY_CELL_VALUE).toString
+        val nodeUniqueId = DataElementDiscoveryUtils.nodeIdentifier(
+          path,
+          sqlNode.code,
+          AuditReportConstants.ELEMENT_DISCOVERY_NODE_TYPE_SQL_NODE,
+          lineNumber
+        )
+        val sourceRuleId = sqlNode.tag
+          .nameExact(Constants.id)
+          .value
+          .headOption
+          .getOrElse(AuditReportConstants.AUDIT_EMPTY_CELL_VALUE)
+
+        workbookResult += List(
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          path,
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          sqlNode.code,
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          sourceRuleId.startsWith("Data.Sensitive.").toString,
+          sourceRuleId,
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          AuditReportConstants.AUDIT_EMPTY_CELL_VALUE,
+          lineNumber,
+          nodeUniqueId,
+          AuditReportConstants.ELEMENT_DISCOVERY_NODE_TYPE_SQL_NODE
+        )
+
+        logger.info("Appended SQL nodes to the workbook")
```

Comment:
also use statsRecoder log message.

Reply:
From our conversation: DED does not use stats recorder and requiring it would need changes outside this PR's scope.